<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRecordAudioClass</class>
 <widget class="QMainWindow" name="QRecordAudioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRecordAudio</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_showspec">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>showspec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_SART_RECORD_VIDEO">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>start_record_video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_STOP_RECORD_VIDEO">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>stop_record_video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_SDLPLAY">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PLAY_VIDEO">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>playvideo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_ENCODE_H264">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>encode_h264</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_DECODE_H264">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>decode_h264</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QRecordAudio.qrc"/>
 </resources>
 <connections/>
</ui>
